新着メッセージコンポーネントの作成

1daLearn opened the pull request:
## チェックリスト

- [x] マージ元のブランチが正しいか(自分の作成したブランチか)
- [x] マージ先のブランチが正しいか(基本はmain)
- [x] 不必要なコードを削除する(コマンドラインからPushする場合)
- [ ] lintを実行する（npm run format）
- [x] プルリクエストの名前を変更する
- [x] 下の欄を埋める

# 概要
<img width="610" alt="スクリーンショット 2023-08-12 15 07 10" src="https://github.com/prapro-ou/castella-frontend/assets/63241922/4c371f8e-6bbf-416b-b096-6d711a9ad50a">

この青い棒を作成する

### 実装したところ
コンポーネント「ThreadsWithNewMessagesDivider」を取り敢えず実装？
### 懸念点
まだ内容の理解が進んでおらず，どこまで正しくできているのかわからない
### 関連するIssue([Closes #数字]で選択可能)

Wansuko-cmd commented:
以下の点を見てほしい
- 正しくcommitできてなさそう（なんかsubmoduleが追加されてるし、コードがない）
- ブランチ名が分かりづらい(feature/new-message-componentなどの方がよき)
- コミットメッセージが適当感が否めない

1daLearn commented:
> * 正しくcommitできてなさそう（なんかsubmoduleが追加されてるし、コードがない）
このような状態です

PS C:\Users\ー\castella-frontend> git status
On branch feature/new-message-component
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
  (commit or discard the untracked or modified content in submodules)
        modified:   castella-frontend (untracked content)

no changes added to commit (use "git add" and/or "git commit -a")
PS C:\Users\ー\castella-frontend> git add .
PS C:\Users\ー\castella-frontend> git status
On branch feature/new-message-component
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
  (commit or discard the untracked or modified content in submodules)
        modified:   castella-frontend (untracked content)

no changes added to commit (use "git add" and/or "git commit -a")
